src/container/__tests__/RelayContainer-test.js

```diff
@@ -563,6 +564,22 @@ describe('RelayContainer', function() {
     });
   });
 
+  describe('props.relay.subscribe', () => {
+    it('forwards to the underlying RelayEnvironment', () => {
+      const mockSubscription = new RelaySubscription();
+      environment.subscribe = jest.fn();
+      render.mockImplementation(function() {
+        this.props.relay.subscribe(mockSubscription);
+      });
+      RelayTestRenderer.render(
+        () => <MockContainer />,
+        environment,
+        mockRoute
+      );
+      expect(environment.subscribe.mock.calls[0][0]).toBe(mockSubscription);
```

Comment:
`expect(environment.subscribe).toBeCalledWith(mockSubscription);`
Not super important but it simplifies a little